### PR TITLE
Fixes the advanced pinpointer crashing the server

### PR DESCRIPTION
**Pull request by farie82**
**What does this PR do:**
Stops the advanced pinpointer from crashing the server

Fixes: #11883

**Changelog:**
:cl:
fix: The syndicate can't use meta warfare no more. Advanced pinpointers no longer crash the server
/:cl:

### CODE REVIEW

File: code/game/gamemodes/nuclear/pinpointer.dm [hunk 1]
```diff
@@ -165,10 +165,13 @@
...
 					if(!targetitem)
 						return
-					var/list/obj/item/target_candidates = get_all_of_type(item_paths[targetitem], subtypes = TRUE)
-						for(var/obj/item/candidate in target_candidates)
-							if(!is_admin_level(candidate.loc.z))
-								target = candidate
+					
+					var/list/target_candidates = get_all_of_type(item_paths[targetitem], subtypes = TRUE)
+					for(var/obj/item/candidate in target_candidates)
+						if(!is_admin_level(candidate.loc.z))
```
Comment (Ty-Omaha) [inline code/game/gamemodes/nuclear/pinpointer.dm]: shouldn't this use get_turf anyways instead?

Comment (Couls) [reply code/game/gamemodes/nuclear/pinpointer.dm]: get_turf wouldn't give you the z level

Comment (farie82) [reply code/game/gamemodes/nuclear/pinpointer.dm]: Damn you just found another bug that caused documents and the like not to show up on the pinpointer

Comment (farie82) [reply code/game/gamemodes/nuclear/pinpointer.dm]: Update. Doesn't seem to have fixed that

**Pull request merged**